plugins/woocommerce-admin/client/products/fields/attribute-input-field/attribute-input-field.tsx

```diff
@@ -16,24 +16,28 @@ import {
 	__experimentalSelectControlMenuItem as MenuItem,
 } from '@woocommerce/components';
 
+type NarrowedQueryAttribute = Pick< QueryProductAttribute, 'id' | 'name' >;
+
 type AttributeInputFieldProps = {
-	value?: ProductAttribute;
+	value?: Pick< QueryProductAttribute, 'id' | 'name' > | null;
 	onChange: (
 		value?: Omit< ProductAttribute, 'position' | 'visible' | 'variation' >
 	) => void;
 	label?: string;
 	placeholder?: string;
 	disabled?: boolean;
-	filteredAttributeIds?: number[];
+	ignoredAttributeIds?: number[];
+	onlyAttributeIds?: number[];
```

Comment:
Why do we need the `onlyAttributeIds` prop? shouldn't this still work like a normal typeahead and thus search all the attributes?

Reply:
Maybe this is an assumption, but I believe the behavior we want in the edit modal in particular is to _only_ have those attributes that already exist on the product be available. Otherwise we'd be adding attributes in the edit modal?

If we don't include a way to limit the attributes available in this component it will end up showing the list of attributes in the entire store, and not just on the product (this is the behavior if I take this away).

Reply:
Hmmm gotcha, after reading the acceptance criteria I could see how this could be a bit ambiguous: `The Attribute field is a searchable dropdown. When clicked, we only show a list of existing attributes.`.
I would of expected the other, where we show all the existing attributes, but will not allow a user to create a new one, but can see how you made your decision as well.

@jarekmorawski could you help clarify this for us? Should this dropdown only show the attributes that are part of the product? or any global attribute?